src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -732,11 +732,15 @@
     <Field type="long" name="pid" label="Process Identifier" />
      </Event>
 
-  <Event name="OSInformation" category="Operating System" label="OS Information" period="endChunk">
+  <Event name="OSInformation" category="Operating System" label="OS Information"
+         description="Description of the OS the JVM runs on, e.g. for example a uname-like output"
```

Comment:
Should it be both "e.g." and "for example"?

Reply:
Thanks Erik, I adjusted the description.